cli/tests/e2e/test_baseline.py

```diff
@@ -125,6 +126,27 @@ def run_sentinel_scan(check: bool = True, base_commit: Optional[str] = None):
         raise e
 
 
+def assert_out_match(snapshot, output, snapshot_name):
+    return snapshot.assert_match(textwrap.dedent(output.stdout), snapshot_name)
+
+
+def assert_err_match(snapshot, output, snapshot_name, replace_base_commit=None):
+    if "PYTEST_USE_OSEMGREP" in os.environ:
```

Comment:
ok, but it would be good to at least check that we still return an error code in osemgrep.
We should add a separate assert that make sure the exit code for pysemgrep (and osemgrep) is
an error for those baseline tests.

Reply:
Done. The exit code is already checked when the engine has completed with errors. For example, https://github.com/returntocorp/semgrep/blob/86250ee8a05de978087061ce60bc2de7937da71e/cli/tests/e2e/test_baseline.py#L712